source/blog/036-rfc-embedded-protocol-2.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Request for Comments: New Embedded Protocol"
+author: Natalie Weizenbaum
+date: 2023-05-19 16:00 PST
+---
+
+If you're not an author of a host package for the Embedded Sass Protocol, you
+can skip this blog post—although if you're a big enough nerd, you may find it
+interesting regardless!
+
+We're planning to make a number of breaking changes to the Embedded Sass
+Protocol, and we want your feedback before we lock in the new way of doing
+things. We intend to make a number of breaking changes all at once to keep the
+total number of disruptions to a minimum.
+
+We're planning two major breaking changes:
+
+1. The Dart Sass embedded host will no longer be released as a separate
+   executable. It will now be bundled into the main Dart Sass executable,
+   accessible by running `sass --embedded`.
+
+2. Every packet in the embedded protocol now includes a compilation ID as part
+   of the packet structure, rather than declaring it in the protocol buffer
+   definitions.
+
+We're using this opportunity to also introduce three much smaller breaking
+changes:
+
+1. The specification for the embedded protocol and the protocol buffer
+   definition have been moved to [the Sass language repository] so that they can
+   be updated at the same time as changes to the language and the JS API.
+
+   [the Sass language repository]: https://github.com/sass/sass/blob/main/spec/embedded-protocol.md
+
+2. The embedded protocol now explicitly declares fields as `optional` using the
```

Comment:
Nit:
> The embedded protocol now explicitly declares which fields are `optional` ..

Or something like that? Otherwise when I read it it comes across as if "everything is optional now"